Site/package.json

```diff
@@ -9,16 +9,17 @@
   "author": "",
   "license": "Apache-2.0",
   "dependencies": {
-    "@veupathdb/components": "^0.11.11",
-    "@veupathdb/eda": "^1.1.51",
+    "@veupathdb/components": "^0.11.13",
+    "@veupathdb/eda": "^1.3.0",
     "@veupathdb/wdk-client": "^0.5.4",
     "@veupathdb/web-common": "^0.2.1"
   },
   "resolutions": {
     "react": "16",
     "react-dom": "16",
     "@types/react": "16",
-    "@types/react-dom": "16"
+    "@types/react-dom": "16",
+    "@veupathdb/core-components": "^0.8.1"
```

Comment:
This is fine! Hopefully we can find a way to avoid this in the future

Reply:
Context for why I did this: yarn was "hoisting" an old version of `@veupathdb/core-components` (0.4.6) that `@veupathdb/components` depends on, and this was causing runtime errors whenever `@veupathdb/eda` would try to import the new "sharing" modals. An alternative solution would have been to upgrade `@veupathdb/core-components` in `@veupathdb/components`, and then to upgrade `@veupathdb/components` in all the `@veupathdb` npm modules.

I think in the long term, we'll want to make `@veupathdb/core-components` a {peer,dev}Dependency in all the `@veupathdb` npm modules that require it. `@veupathdb/core-components` is defining some React context, so we should only include a single version in our site bundles.